Dockerfile

```diff
@@ -17,8 +17,3 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local
 
 RUN pecl install xdebug \
     && docker-php-ext-enable xdebug
-
-COPY composer.* ./
-RUN composer install $COMPOSER_FLAGS --no-scripts --no-autoloader
-COPY . .
-RUN composer install $COMPOSER_FLAGS
```

Comment:
Jeste jsem prihodil tohle (+ patricna uprava `docker-compose`). Je to libka, takze neni duvod mi Composer packages v containeru. Jen to zbytecne komplikuje, kdyz delas nejaky updaty (nezbuildis image, aby si fixnul verze packages, protoze se ti nenainstalujou packages :slightly_smiling_face: )

Reply:
to byl asi copy paste